src/start-client.jsx

```diff
@@ -12,8 +12,13 @@ import '~/theme';
 import configureStore from '@plone/volto/store';
 import { Api, persistAuthToken, ScrollToTop } from '@plone/volto/helpers';
 
+import * as Sentry from '@sentry/browser';
+import initSentry from './sentry';
+
 export const history = createBrowserHistory();
 
+initSentry(Sentry);
```

Comment:
We call initSentry always? Even if Sentry is not enabled?

Reply:
In initSentry first we check if sentry is enabled, and only if it's enabled we call the Sentry.init(...) what actually initializes it. I wanted to avoid to have similar code in start-server.js and start-client.js, that's why all the checks are done inside initSentry.

Reply:
If we lazy load Sentry, isn't there a risk that some exceptions will not be sent to sentry?

Reply:
Not sure that might happen... @tiberiuichim @pnicolli what do you think?